### PR TITLE
Week 7

### PR DESCRIPTION
**Pull request by twof**

### CODE REVIEW

**Comment by VaporNationTravis** [comment]
### Spell Checker found issues

#### [week7-18-03-12.md](/twof/VaporNation/tree/draft/week7-18-03-12.md)

Line | Typo |
| --- | ------ |
5 | ### HTTP **Pipelining** | 
15 | ### Tanner Nelson On Swift by **Sundell** | 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: week7-18-03-12.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Week 7
+
+## New Features
+
+### Adds Ability To Execute Shell Commands
+#### Will include or not include based on Tanner's response
+
+[vapor/core#100](https://github.com/vapor/core/pull/100)
```
Comment (Yasumoto) [inline week7-18-03-12.md]: I think this is just an extension on the `Process` [in Foundation](https://developer.apple.com/documentation/foundation/process)

Comment (twof) [reply week7-18-03-12.md]: Tanner confirmed that this is new:

https://github.com/vapor/core/pull/100#issuecomment-375071470

Comment (Yasumoto) [reply week7-18-03-12.md]: Sweet! 👍

**Pull request merged**